lesson28/src/js/changepassword.js

```diff
@@ -40,4 +42,19 @@ const togglePasswordDisplay = target => {
     target.classList.toggle("is-open");
 }
 
-submitButton.addEventListener("click", () => console.log("submitされました。この後はまだ未実装です。"))
+const changeRegisteredDataOfPassword = () => {
+    const newPassword = passwordOfInput.value;
```

Comment:
IMO
この時点ではまだ入力中のパスワードの値を参照しているだけなのでnewではないですよね。なので
```suggestion
    const passwordValue = passwordOfInput.value;
```
がわかりやすいと感じました

Reply:
ご指摘ありがとうございます！
こちらもその通りだと思いました。
修正いたしました。
[470db80](https://github.com/haru-programming/JavaScript-lessons/pull/36/commits/470db80f5974642602da5e7fb6241cf9e37e6d56)